### ISSUE TITLE
FIDO2 tests fail for v1.3.1-test.20230414

### ISSUE THREAD

**Issue by robin-nitrokey** [Nitrokey/nitrokey-3-firmware#235]
With v1.3.1-test.20230414, calling GetAssertion for a credential that was created using the same firmware version causes a CTAP error 0x7F (OTHER).

Reproduction:
- `python3 test-release.py v1.3.1-test.20230414` with https://github.com/nitrokey/nitrokey-3-tests using the usbip runner
- `nitropy nk3 test --only fido2` with hardware

Logs:
<details>
<summary>Backtrace from nitrokey-3-tests</summary>

```
________________________________ TestFido2.test ________________________________

self = <fido2.client.Fido2Client object at 0x7f7451d770d0>
options = PublicKeyCredentialRequestOptions(challenge=b'jlL\xe6\xd41O\xebO\xe4u<\xcam\xaao\xe7\x1b\x17\xb6\xb3\xf9\xd8\xc9&;\xad...(\x90', transports=None)], user_verification=<UserVerificationRequirement.DISCOURAGED: 'discouraged'>, extensions=None)
event = <threading.Event object at 0x7f7451d89eb0>

    def get_assertion(
        self,
        options: PublicKeyCredentialRequestOptions,
        event: Optional[Event] = None,
    ) -> AssertionSelection:
        """Get an assertion.
    
        :param options: PublicKeyCredentialRequestOptions data.
        :param threading.Event event: (optional) Signal to abort the operation.
        """
    
        options = PublicKeyCredentialRequestOptions.from_dict(options)
        event = event or Event()
        if options.timeout:
            timer = Timer(options.timeout / 1000, event.set)
            timer.daemon = True
            timer.start()
    
        logger.debug(f"Assert a credential for RP ID: {options.rp_id}")
        self._verify_rp_id(options.rp_id)
    
        client_data = self._build_client_data(
            CollectedClientData.TYPE.GET, options.challenge
        )
    
        try:
>           return self._backend.do_get_assertion(
                client_data,
                options.rp_id,
                options.allow_credentials,
                options.extensions,
                options.user_verification,
                event,
            )

/venv/lib/python3.9/site-packages/fido2/client.py:836: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <fido2.client._Ctap2ClientBackend object at 0x7f7451d77160>
client_data = CollectedClientData(type='webauthn.get', challenge=b'jlL\xe6\xd41O\xebO\xe4u<\xcam\xaao\xe7\x1b\x17\xb6\xb3\xf9\xd8\xc9&;\xad\xd9\x16\xdb+\xdf', origin='https://example.com', cross_origin=False)
rp_id = 'example.com'
allow_list = [PublicKeyCredentialDescriptor(type=<PublicKeyCredentialType.PUBLIC_KEY: 'public-key'>, id=b'\xa3\x00X\x98\xaaP&\xda#\...f4\xae\xf9\x01L\xc3c\xb1\xad\x8b<\x04\xc0\xe8\xc1#\xf0\x02P\xb8\x0bL\x7f6\xd8\xfe\xbb;hi|\xa3z(\x90', transports=None)]
extensions = None
user_verification = <UserVerificationRequirement.DISCOURAGED: 'discouraged'>
event = <threading.Event object at 0x7f7451d89eb0>

    def do_get_assertion(
        self,
        client_data,
        rp_id,
        allow_list,
        extensions,
        user_verification,
        event,
    ):
        if allow_list:
            # Filter out credential IDs which are too long
            max_len = self.info.max_cred_id_length
            if max_len:
                allow_list = [e for e in allow_list if len(e) <= max_len]
            if not allow_list:
                raise CtapError(CtapError.ERR.NO_CREDENTIALS)
    
            # Reject the request if too many credentials remain.
            max_creds = self.info.max_creds_in_list
            if max_creds and len(allow_list) > max_creds:
                raise ClientError.ERR.BAD_REQUEST("allow_list too long")
    
        # Process extensions
        client_inputs = extensions or {}
        extension_inputs = {}
        used_extensions = []
        permissions = ClientPin.PERMISSION(0)
        try:
            for ext in [cls(self.ctap2) for cls in self.extensions]:
                auth_input, req_perms = ext.process_get_input_with_permissions(
                    client_inputs
                )
                if auth_input is not None:
                    used_extensions.append(ext)
                    permissions |= req_perms
                    extension_inputs[ext.NAME] = auth_input
        except ValueError as e:
            raise ClientError.ERR.CONFIGURATION_UNSUPPORTED(e)
    
        on_keepalive = _user_keepalive(self.user_interaction)
    
        pin_protocol, pin_token, pin_auth, internal_uv = self._get_auth_params(
            client_data, rp_id, user_verification, permissions, event, on_keepalive
        )
        options = {"uv": True} if internal_uv else None
    
>       assertions = self.ctap2.get_assertions(
            rp_id,
            client_data.hash,
            allow_list or None,
            extension_inputs or None,
            options,
            pin_auth,
            pin_protocol.VERSION if pin_protocol else None,
            event=event,
            on_keepalive=on_keepalive,
        )

/venv/lib/python3.9/site-packages/fido2/client.py:683: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <fido2.ctap2.base.Ctap2 object at 0x7f7451d77190>
args = ('example.com', b'De\x1e\xca\xc2\x152\x9b\x96[F\xece<p\xfd\xc8J\xab$P\xea\xf8\xad\xfe[\x03\xa4{\x86\xe4F', [PublicKeyC...\xad\x8b<\x04\xc0\xe8\xc1#\xf0\x02P\xb8\x0bL\x7f6\xd8\xfe\xbb;hi|\xa3z(\x90', transports=None)], None, None, None, ...)
kwargs = {'event': <threading.Event object at 0x7f7451d89eb0>, 'on_keepalive': <function _user_keepalive.<locals>.on_keepalive at 0x7f7451e26820>}

    def get_assertions(self, *args, **kwargs) -> List[AssertionResponse]:
        """Convenience method to get list of assertions.
    
        See get_assertion and get_next_assertion for details.
        """
>       first = self.get_assertion(*args, **kwargs)

/venv/lib/python3.9/site-packages/fido2/ctap2/base.py:439: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <fido2.ctap2.base.Ctap2 object at 0x7f7451d77190>, rp_id = 'example.com'
client_data_hash = b'De\x1e\xca\xc2\x152\x9b\x96[F\xece<p\xfd\xc8J\xab$P\xea\xf8\xad\xfe[\x03\xa4{\x86\xe4F'
allow_list = [PublicKeyCredentialDescriptor(type=<PublicKeyCredentialType.PUBLIC_KEY: 'public-key'>, id=b'\xa3\x00X\x98\xaaP&\xda#\...f4\xae\xf9\x01L\xc3c\xb1\xad\x8b<\x04\xc0\xe8\xc1#\xf0\x02P\xb8\x0bL\x7f6\xd8\xfe\xbb;hi|\xa3z(\x90', transports=None)]
extensions = None, options = None, pin_uv_param = None, pin_uv_protocol = None

    def get_assertion(
        self,
        rp_id: str,
        client_data_hash: bytes,
        allow_list: Optional[List[Mapping[str, Any]]] = None,
        extensions: Optional[Mapping[str, Any]] = None,
        options: Optional[Mapping[str, Any]] = None,
        pin_uv_param: Optional[bytes] = None,
        pin_uv_protocol: Optional[int] = None,
        *,
        event: Optional[Event] = None,
        on_keepalive: Optional[Callable[[int], None]] = None,
    ) -> AssertionResponse:
        """CTAP2 getAssertion command.
    
        :param rp_id: The RP ID of the credential.
        :param client_data_hash: SHA256 hash of the ClientData used.
        :param allow_list: Optional list of PublicKeyCredentialDescriptors.
        :param extensions: Optional dict of extensions.
        :param options: Optional dict of options.
        :param pin_uv_param: Optional PIN/UV auth parameter.
        :param pin_uv_protocol: The version of PIN/UV protocol used, if any.
        :param event: Optional threading.Event used to cancel the request.
        :param on_keepalive: Optional callback function to handle keep-alive messages
            from the authenticator.
        :return: The new assertion.
        """
        logger.debug("Calling CTAP2 get_assertion")
        return AssertionResponse.from_dict(
>           self.send_cbor(
                Ctap2.CMD.GET_ASSERTION,
                args(
                    rp_id,
                    client_data_hash,
                    allow_list,
                    extensions,
                    options,
                    pin_uv_param,
                    pin_uv_protocol,
                ),
                event=event,
                on_keepalive=on_keepalive,
            )
        )

/venv/lib/python3.9/site-packages/fido2/ctap2/base.py:411: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <fido2.ctap2.base.Ctap2 object at 0x7f7451d77190>
cmd = <CMD.GET_ASSERTION: 2>
data = {1: 'example.com', 2: b'De\x1e\xca\xc2\x152\x9b\x96[F\xece<p\xfd\xc8J\xab$P\xea\xf8\xad\xfe[\x03\xa4{\x86\xe4F', 3: [P...4\xae\xf9\x01L\xc3c\xb1\xad\x8b<\x04\xc0\xe8\xc1#\xf0\x02P\xb8\x0bL\x7f6\xd8\xfe\xbb;hi|\xa3z(\x90', transports=None)]}

    def send_cbor(
        self,
        cmd: int,
        data: Optional[Mapping[int, Any]] = None,
        *,
        event: Optional[Event] = None,
        on_keepalive: Optional[Callable[[int], None]] = None,
    ) -> Mapping[int, Any]:
        """Sends a CBOR message to the device, and waits for a response.
    
        :param cmd: The command byte of the request.
        :param data: The payload to send (to be CBOR encoded).
        :param event: Optional threading.Event used to cancel the request.
        :param on_keepalive: Optional function called when keep-alive is sent by
            the authenticator.
        """
        request = struct.pack(">B", cmd)
        if data is not None:
            request += cbor.encode(data)
        response = self.device.call(CTAPHID.CBOR, request, event, on_keepalive)
        status = response[0]
        if status != 0x00:
>           raise CtapError(status)
E           fido2.ctap.CtapError: CTAP error: 0x7F - OTHER

/venv/lib/python3.9/site-packages/fido2/ctap2/base.py:241: CtapError

During handling of the above exception, another exception occurred:

self = <tests.basic.TestFido2 object at 0x7f7451e20970>
device = <utils.device.UsbipDevice object at 0x7f7451e2de80>

    def test(self, device: Device) -> None:
>       self.run(device)

utils/upgrade.py:74: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
utils/upgrade.py:45: in run
    self.verify(context, state)
tests/basic.py:45: in verify
    fido2.authenticate([credential])
utils/fido2.py:95: in authenticate
    get_assertion_result = self.client.get_assertion(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <fido2.client.Fido2Client object at 0x7f7451d770d0>
options = PublicKeyCredentialRequestOptions(challenge=b'jlL\xe6\xd41O\xebO\xe4u<\xcam\xaao\xe7\x1b\x17\xb6\xb3\xf9\xd8\xc9&;\xad...(\x90', transports=None)], user_verification=<UserVerificationRequirement.DISCOURAGED: 'discouraged'>, extensions=None)
event = <threading.Event object at 0x7f7451d89eb0>

    def get_assertion(
        self,
        options: PublicKeyCredentialRequestOptions,
        event: Optional[Event] = None,
    ) -> AssertionSelection:
        """Get an assertion.
    
        :param options: PublicKeyCredentialRequestOptions data.
        :param threading.Event event: (optional) Signal to abort the operation.
        """
    
        options = PublicKeyCredentialRequestOptions.from_dict(options)
        event = event or Event()
        if options.timeout:
            timer = Timer(options.timeout / 1000, event.set)
            timer.daemon = True
            timer.start()
    
        logger.debug(f"Assert a credential for RP ID: {options.rp_id}")
        self._verify_rp_id(options.rp_id)
    
        client_data = self._build_client_data(
            CollectedClientData.TYPE.GET, options.challenge
        )
    
        try:
            return self._backend.do_get_assertion(
                client_data,
                options.rp_id,
                options.allow_credentials,
                options.extensions,
                options.user_verification,
                event,
            )
        except CtapError as e:
>           raise _ctap2client_err(e)
E           fido2.client.ClientError: (<ERR.OTHER_ERROR: 1>, CtapError('CTAP error: 0x7F - OTHER'))
```
</details>

<details>
<summary>Runner logs from nitrokey-3-tests</summary>

```
 INFO  usbd_ctaphid::pipe    > >> 
 INFO  usbd_ctaphid::pipe    > FF FF FF FF 86 00 08 CC 06 BB 4C CD 84 50 B8 00
 INFO  usbd_ctaphid::pipe    > init
 INFO  usbd_ctaphid::pipe    > >> 
 INFO  usbd_ctaphid::pipe    > 00 00 00 02 90 00 01 04 00 00 00 00 00 00 00 00
 INFO  usbd_ctaphid::pipe    > init
 INFO  usbip_runner          > Set status: Processing
 INFO  usbip_runner          > Set status: Idle
 INFO  fido_authenticator::state > err loading: Other
 INFO  fido_authenticator::state > error with previous state! Other
 INFO  usbip_runner              > Set status: Processing
 INFO  usbip_runner              > Set status: Idle
 INFO  usbip_runner              > Set status: Processing
 INFO  usbip_runner              > Set status: Idle
 INFO  !                         > found aaguid
 INFO  usbd_ctaphid::pipe        > Got 159 bytes response from authenticator, starting send
 INFO  usbd_ctaphid::pipe        > >> 
 INFO  usbd_ctaphid::pipe        > 00 00 00 02 90 00 01 04 00 00 00 00 00 00 00 00
 INFO  usbd_ctaphid::pipe        > init
 INFO  usbip_runner              > Set status: Processing
 INFO  usbip_runner              > Set status: Idle
 INFO  usbip_runner              > Set status: Processing
 INFO  usbip_runner              > Set status: Idle
 INFO  !                         > found aaguid
 INFO  usbd_ctaphid::pipe        > Got 159 bytes response from authenticator, starting send
 INFO  usbd_ctaphid::pipe        > >> 
 INFO  usbd_ctaphid::pipe        > 00 00 00 02 90 00 EB 01 A4 01 58 20 33 85 77 F6
 INFO  usbd_ctaphid::pipe        > init
 INFO  usbd_ctaphid::pipe        > >> 
 INFO  usbd_ctaphid::pipe        > 00 00 00 02 00 6D 65 6A 45 78 61 6D 70 6C 65 20
 INFO  usbd_ctaphid::pipe        > >> 
 INFO  usbd_ctaphid::pipe        > 00 00 00 02 01 2D 6B 65 79 A2 63 61 6C 67 27 64
 INFO  usbd_ctaphid::pipe        > >> 
 INFO  usbd_ctaphid::pipe        > 00 00 00 02 02 70 65 6A 70 75 62 6C 69 63 2D 6B
 INFO  usbd_ctaphid::pipe        > >> 
 INFO  usbd_ctaphid::pipe        > 00 00 00 02 03 79 00 00 00 00 00 00 00 00 00 00
 INFO  usbip_runner              > Set status: Processing
 INFO  usbip_runner              > Set status: Idle
 INFO  !                         > algo: -8
 INFO  !                         > MC options: None
 INFO  usbip_runner              > Set status: Processing
 INFO  usbip_runner              > Set status: WaitingForUserPresence
 INFO  usbip_runner              > >>>> Received confirmation request. Confirming automatically.
 INFO  usbip_runner              > Set status: Idle
 INFO  usbip_runner              > Set status: Idle
 INFO  usbip_runner              > Set status: Processing
 INFO  usbip_runner              > Set status: Idle
 INFO  usbip_runner              > Set status: Processing
 INFO  usbip_runner              > Set status: Idle
 INFO  usbip_runner              > Set status: Processing
 INFO  usbip_runner              > Set status: Idle
 INFO  usbip_runner              > Set status: Processing
 INFO  usbip_runner              > Set status: Idle
 INFO  !                         > deleted public Ed25519 key: true
 INFO  usbip_runner              > Set status: Processing
 INFO  usbip_runner              > Set status: Idle
 INFO  usbip_runner              > Set status: Processing
 INFO  usbip_runner              > Set status: Idle
 INFO  usbip_runner              > Set status: Processing
 INFO  usbip_runner              > Set status: Idle
 INFO  usbip_runner              > Set status: Processing
 INFO  usbip_runner              > Set status: Idle
 INFO  !                         > nonce = [195, 99, 177, 173, 139, 60, 4, 192, 232, 193, 35, 240]
 INFO  usbip_runner              > Set status: Processing
 INFO  usbip_runner              > Set status: Idle
 INFO  usbip_runner              > Set status: Processing
 INFO  usbip_runner              > Set status: Idle
 INFO  usbip_runner              > Set status: Processing
 INFO  usbip_runner              > Set status: Idle
 INFO  fido_authenticator::credential > credential for algorithm -8
 INFO  !                              > :: stripping ID
 INFO  usbip_runner                   > Set status: Processing
 INFO  usbip_runner                   > Set status: Idle
 INFO  !                              > MC created cred id
 INFO  usbip_runner                   > Set status: Processing
 INFO  usbip_runner                   > Set status: Idle
 INFO  usbip_runner                   > Set status: Processing
 INFO  usbip_runner                   > Set status: Idle
 INFO  !                              > found aaguid
 INFO  usbip_runner                   > Set status: Processing
 INFO  usbip_runner                   > Set status: Idle
 INFO  usbip_runner                   > Set status: Processing
 INFO  usbip_runner                   > Set status: Idle
 INFO  usbip_runner                   > Set status: Processing
 INFO  usbip_runner                   > Set status: Idle
 INFO  !                              > deleted private credential key: true
 INFO  usbd_ctaphid::pipe             > Got 976 bytes response from authenticator, starting send
 INFO  usbd_ctaphid::pipe             > >> 
 INFO  usbd_ctaphid::pipe             > 00 00 00 02 90 00 01 04 00 00 00 00 00 00 00 00
 INFO  usbd_ctaphid::pipe             > init
 INFO  usbip_runner                   > Set status: Processing
 INFO  usbip_runner                   > Set status: Idle
 INFO  usbip_runner                   > Set status: Processing
 INFO  usbip_runner                   > Set status: Idle
 INFO  !                              > found aaguid
 INFO  usbd_ctaphid::pipe             > Got 159 bytes response from authenticator, starting send
 INFO  usbd_ctaphid::pipe             > >> 
 INFO  usbd_ctaphid::pipe             > 00 00 00 02 90 01 06 02 A3 01 6B 65 78 61 6D 70
 INFO  usbd_ctaphid::pipe             > init
 INFO  usbd_ctaphid::pipe             > >> 
 INFO  usbd_ctaphid::pipe             > 00 00 00 02 00 BC A3 00 58 98 AA 50 26 DA 23 88
 INFO  usbd_ctaphid::pipe             > >> 
 INFO  usbd_ctaphid::pipe             > 00 00 00 02 01 04 EE 56 6D 25 03 4E 60 0B 44 16
 INFO  usbd_ctaphid::pipe             > >> 
 INFO  usbd_ctaphid::pipe             > 00 00 00 02 02 AB 2D 13 C4 4F B8 C6 54 E8 79 87
 INFO  usbd_ctaphid::pipe             > >> 
 INFO  usbd_ctaphid::pipe             > 00 00 00 02 03 36 D8 FE BB 3B 68 69 7C A3 7A 28
 INFO  usbip_runner                   > Set status: Processing
 INFO  usbip_runner                   > Set status: Idle
 INFO  usbip_runner                   > Set status: Processing
 INFO  usbip_runner                   > Set status: Idle
 INFO  !                              > found 1 applicable credentials
 INFO  !                              > Credential { ctap: Fido21Pre, data: CredentialData { rp: PublicKeyCredentialRpEntity { id: "example.com", name: None, url: None }, user: PublicKeyCredentialUserEntity { id: b'user_id', icon: None, name: None, display_name: None }, creation_time: 0, use_counter: true, algorithm: -8, key: WrappedKey(b'\x03\x00\x00\x00\x00\x00\x00\x00\x00$\xda\xab\x07H\xc72w\xe9.\x8a\xee^\xd8\x1e\xa1\xaf\xdd\xc3@\xd5\xc2\xdd#Y\xfc\xf9\xbc{\xdc\xa47+\x83Z-\xe5\x01\x00\x00\x00\x00\x00\x00\x00\x0c\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02\x00\x00\x00\x00\x00\x00\x00\x100\x93\xb9\xb2dC\x1e\x8dU\t\xfe-\\4e\x99'), hmac_secret: None, cred_protect: None }, nonce: b'\xc3c\xb1\xad\x8b<\x04\xc0\xe8\xc1#\xf0' }
 INFO  usbip_runner                   > Set status: Processing
 INFO  usbip_runner                   > Set status: Idle
 INFO  !                              > asking for up
 INFO  usbip_runner                   > Set status: Processing
 INFO  usbip_runner                   > Set status: WaitingForUserPresence
 INFO  usbip_runner                   > >>>> Received confirmation request. Confirming automatically.
 INFO  usbip_runner                   > Set status: Idle
 INFO  usbip_runner                   > Set status: Idle
 INFO  usbip_runner                   > Set status: Processing
 INFO  usbip_runner                   > Set status: Idle
 INFO  !                              > key result
 INFO  usbd_ctaphid::pipe             > Got 1 bytes response from authenticator, starting send
```
</details>

<details>
<summary>Client logs from nitrokey-3-tests</summary>

```
DEBUG    fido2.server:server.py:153 Fido2Server initialized for RP: PublicKeyCredentialRpEntity(name='Example RP', id='example.com')
DEBUG    fido2.server:server.py:187 Starting new registration, existing credentials: 
DEBUG    fido2.client:client.py:772 Register a new credential for RP ID: example.com
DEBUG    fido2.ctap2.base:base.py:361 Calling CTAP2 make_credential
DEBUG    fido2.server:server.py:290 Verifying attestation of type packed
INFO     fido2.server:server.py:297 New credential registered: a3005898aa5026da23889da587cd33b2f3345270557326d2cfd41c836f50057ba636bb2693401d5a8d78ce5f023b4c28e2b3c2c074f4a622264904ee566d25034e600b44167bbd18e993a24d9483045bc1ee5599f45badd37573527de825ba0715cbcd61378de7ac4e947bde02ab813f312eecc15fab2d13c44fb8c654e879873474b762cf402d0eca7ee8a6adfe98edf6e1a3e0b7b4890f46f4aef9014cc363b1ad8b3c04c0e8c123f00250b80b4c7f36d8febb3b68697ca37a2890
DEBUG    fido2.server:server.py:329 Starting new authentication, for credentials: a3005898aa5026da23889da587cd33b2f3345270557326d2cfd41c836f50057ba636bb2693401d5a8d78ce5f023b4c28e2b3c2c074f4a622264904ee566d25034e600b44167bbd18e993a24d9483045bc1ee5599f45badd37573527de825ba0715cbcd61378de7ac4e947bde02ab813f312eecc15fab2d13c44fb8c654e879873474b762cf402d0eca7ee8a6adfe98edf6e1a3e0b7b4890f46f4aef9014cc363b1ad8b3c04c0e8c123f00250b80b4c7f36d8febb3b68697ca37a2890
DEBUG    fido2.client:client.py:828 Assert a credential for RP ID: example.com
DEBUG    fido2.ctap2.base:base.py:409 Calling CTAP2 get_assertion
```
</details>

Observations:
- Credentials that are created with an older firmware version still work.
- Resident credentials are not affected.

**Comment by robin-nitrokey** [comment]
It looks like this is caused by https://github.com/Nitrokey/trussed/pull/21.  The key is wrapped here with the `rp_id_hash` as associated data:

https://github.com/solokeys/fido-authenticator/blob/0.1.1/src/ctap2.rs#L323-328

But it is unwrapped with empty associated data:

https://github.com/solokeys/fido-authenticator/blob/0.1.1/src/ctap2.rs#L1462-L1480

In old firmware versions, the associated data was ignored when wrapping a key.  Since trussed v0.1.0-nitrokey.9, this is no longer the case.  So we can unwrap old credentials without associated data but fail to unwrap new credentials with associated data.  The easiest fix would be to change `fido-authenticator` to wrap with empty associated data.

**Comment by sosthene-nitrokey** [comment]
https://github.com/solokeys/fido-authenticator/pull/24

There's already [this PR](https://github.com/solokeys/fido-authenticator/pull/24) implementing the required fix.

**Comment by sosthene-nitrokey** [comment]
Fixed by #236
